### PR TITLE
chore: add estimated USD value to swap analytics events

### PR DESCRIPTION
**Pull request by silasbw**
### Test plan

Verify properties in analytics backends.

### CODE REVIEW

File: src/analytics/Properties.tsx [hunk 1]
```diff
@@ -1257,13 +1257,15 @@ interface SwapEventsProperties {
       fromTokenBalance: string
       swapExecuteTxId: string
       swapApproveTxId: string
+      estimatedUsdValue?: number
```
Comment (dievazqu) [inline src/analytics/Properties.tsx:1260]: nit: Do you think it makes sense to include this field in the `SwapQuoteEvent`?

File: src/swap/saga.ts [hunk 2]
```diff
@@ -162,6 +181,7 @@ export function* swapSubmitSaga(action: PayloadAction<SwapInfo>) {
     ValoraAnalytics.track(SwapEvents.swap_execute_success, {
       ...defaultSwapExecuteProps,
       ...timeMetrics,
+      estimatedUsdValue,
```
Comment (dievazqu) [inline src/swap/saga.ts:184]: nit: I think you can include this value in the `defaultSwapExecuteProps`

File: src/swap/saga.ts [hunk 3]
```diff
@@ -79,6 +94,10 @@ export function* swapSubmitSaga(action: PayloadAction<SwapInfo>) {
     ? fromToken.balance.shiftedBy(fromToken.decimals).toString()
     : ''
 
+  const estimatedUsdValue = fromToken
```
Comment (dievazqu) [inline src/swap/saga.ts:97]: What do you think about having 2 fields, one for the estimated usd value for the selling token, and another for the buying token?

Something like:
```
const estimatedUsdValueSellingToken = calculateEstimatedUsdValue({ tokenInfo: fromToken, tokenAmount: sellAmount })
const estimatedUsdValueBuyingToken = calculateEstimatedUsdValue({ tokenInfo: toToken, tokenAmount: buyAmount })
```

In theory, `fromToken`, `toToken`, `sellAmount` and `buyAmount` will be defined. So I don't know if we need to check them.
If they aren't defined we will have errors in other parts as well.

**Review by dievazqu — APPROVED** [review]
LGTM!

File: src/analytics/Properties.tsx (hunk 1 above)
Comment (silasbw) [reply src/analytics/Properties.tsx:1260]: yeah, probably.  I was going to chat with you about clarifying some of the other fields (e.g., `.usdValue`) before adding this one to `SwapQuoteEvent` (and by implication `swap_review_submit`).

File: src/swap/saga.ts (hunk 3 above)
Comment (silasbw) [reply src/swap/saga.ts:97]: :+1:

**Pull request merged**